json_encoding.rst

```diff
@@ -0,0 +1,62 @@
+#########################
+Raiden transport messages
+#########################
+
+Transport
+=========
+Messages are exchanged in JSON format using Matrix as a transport layer. Messages can be sent either via broadcast channel or via direct messaging
```

Comment:
Why should the transport layer be specified here? The messages are indifferent to the transport used.